### ISSUE TITLE
"TypeError: Cannot read property 'screen' of undefined" with Vue CLI 3 plugin

### ISSUE THREAD

**Issue by hugetiny** [quasarframework/quasar#5705]
**Describe the bug**
I want to embeding quasar to an existing electron-vue project.
Got "TypeError: Cannot read property 'screen' of undefined" when rendering Qlayout

**Codepen/jsFiddle/Codesandbox (required)**
An existing electron-vue project
https://github.com/SimulatedGREG/electron-vue

**To Reproduce**
1. Create an electron-vue project
2. vue add quasar
3. node .electron-vue/dev-runner.js

**Expected behavior**
No errors with Quasar component

**Screenshots**
![image](https://user-images.githubusercontent.com/19659202/69696554-f70f0500-111a-11ea-81da-f315133dec90.png)

**Platform (please complete the following information):**
OS:Windows10
Node:10.16.3
NPM:6.9.0
Yarn:1.19
Browsers:Chrome
iOS:No
Android:No
Electron:Yes

**Comment by emahuni** [comment]
what was the solution to this? I am having the same issue. Exactly! happened when I upgraded quasar app 2 and quasar 1.12.6.

**Comment by haizad** [comment]
Got same issue.

**Comment by whoacowboy** [comment]
@hugetiny 

I am having the same issue and I was able to resolve it for now. 

The default installation worked for me for a number of compiles then randomly it stopped working. 

I removed the `quasar` package from the `ui/node_modules` and added it to `ui/dev/node_modules` and it worked. 

I have no idea why this fixed it.

I did try all the above solutions and they didn't work for me. It would be good to figure out how to get this working with the `quasar` package in `ui/node_modules` because no I have to re jigger all of my tests.
